testnets/lombardledgertestnet/chain.json

```diff
@@ -0,0 +1,53 @@
+{
+  "$schema": "../../chain.schema.json",
+  "chain_name": "lombardledgertestnet",
```

Comment:
I think chain_name `lombardtestnet` makes more sense to use. Then the mainnet will use chain_name `lombard`. Am I wrong?

Reply:
note that how it's shown to use will go by pretty_name. (users will never see this chain_name). Also, changing what it is currently isn't required--just thought I'd point it out

Reply:
> note that how it's shown to use will go by pretty_name (Where is `"pretty_name"`?). (users will never see this chain_name). Also, changing what it is currently isn't required--just thought I'd point it out

Reply:
Checked internally, `lombardledgertestnet` is ok for us. I added the `pretty_name` field